manifests/k/KPN/HIPINv4/4.3.2.6316/KPN.HIPINv4.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: KPN.HIPINv4

```

```suggestion
PackageIdentifier: KPN.HIPIN
```
Versions generally do not need to be included in the package identifier